proxy/src/main/java/com/wavefront/agent/handlers/AbstractReportableEntityHandler.java

```diff
@@ -34,6 +37,7 @@
   private static final Logger logger = Logger.getLogger(
       AbstractReportableEntityHandler.class.getCanonicalName());
   protected static final MetricsRegistry LOCAL_REGISTRY = new MetricsRegistry();
+  protected static final String MULTICASTING_TENANT_TAG_KEY = "multicastingTenantName";
```

Comment:
maybe this should be configurable

Reply:
Agree on this. Will add this in future refactoring process.